.github/workflows/crowdin-update-resources.yml

```diff
@@ -22,6 +22,6 @@ jobs:
     - name: Crowdin Update
       run: npm run crowdin-update
       env:
-        CROWDIN_PROJECT_ID: appium-desktop
-        CROWDIN_PROJECT_KEY: ${{ secrets.CROWDIN_PROJECT_KEY }}
-      
+        # appium-desktop
+        CROWDIN_PROJECT_ID: 346705
+        CROWDIN_TOKEN: ${{ secrets.CROWDIN_TOKEN }}
```

Comment:
Do you use this from your api token, or from 1pass word (the one I previously created, which ties to my account)
Just want to remove the token from 1pass if that was not used to reduce confusion

Reply:
The v2 API uses a new token format. I have added it to the `appium` account at crowdin and to github secrets.

Reply:
Yes, i added the new format's one before (maybe last month or so), so to avoid duplications. Let me delete mine from the 1pass, that has `Crowdin's v2 access token. It ties with a personal access token by Kazu` in the note

Reply:
btw, in the Appium vaults?

Reply:
(deleted mine)

Reply:
> btw, in the Appium vaults?

I don't have 1pass on my machine
I could send the token to you, so you could update 1pass

Reply:
Oh, i see. either is ok for me, or I can send an invitation for you. I wanted to clarify where the token was.